src/domain/messages/form/MessageForm.tsx

```diff
@@ -102,7 +102,7 @@ const MessageForm = ({ protocol, ...delegatedProps }: Props) => {
 
     // When the user sets recipient as all, event and occurrence
     // choices are reset
-    if (value === 'ALL') {
+    if (['ALL', 'INVITED'].includes(value)) {
```

Comment:
Now the comment and code are out-of-sync with each other. Is the comment really necessary? What is the value it brings? For example renaming `const value = event?.target.value;` to `const recipient = event?.target.value;` and changing this it `if (['ALL', 'INVITED'].includes(recipient)) {` could convey largely the same information IMHO.